### ISSUE TITLE
[Messaging] Sanitize message for rendering in a conversation

### ISSUE THREAD

**Issue by l2succes** [artsy/metaphysics#648]

**Comment by alloy** [comment]
Shouldn’t this be done in Impulse? I think @joeyAghion even mentioned at some point that SendGrid (?) might provide a library to perform common sanitizations.

Also, our of curiosity, do we have examples yet of what the rendered messages look like before sanitization?

**Comment by l2succes** [comment]
@alloy Yes I think we could potentially do this in Impulse. Unless it means we'll be tempering with the partner's CMS experience who actually should see the context of the collector submitting the inquiry.

**Comment by alloy** [comment]
Good point, but that could just be separated from each other. Ideally we wouldn’t even add the “about this collector” stuff in the first place.
